.github/workflows/release.yaml

```diff
@@ -0,0 +1,20 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish-npm:
```

Comment:
we shouldn't publish a release for a commit if CI is failing. Can we keep this in the CI workflow to maintain that dependency?

Reply:
I assumed we would rely on a shared moral code to not create a tag if CI fails.
That may be a bit weak :P

Reply:
So long as we push the commit and let CI run before pushing the tag that should work.

The general gotcha is that cutting a release requires updating the version in the package.json and lock file. So there is new code in the release (hopefully nothing more than the version bump)